src/content/en/updates/2017/07/chrome-61-media-updates.md

```diff
@@ -0,0 +1,101 @@
+
+To improve battery life, Chrome now disables video tracks when the video is
+played in the background if the video uses [Media Source Extensions (MSE)].
+
+You can inspect these changes by going to the `chrome://media-internals` page,
+and filter for the "info" property. When the tab containing a playing video
+becomes inactive, you'll see a message like `Selected video track: []`
+indicating that the video track has been disabled. When tab becomes active
+again, video track is re-enabled automatically.
+
+<figure>
+  <img src="/web/updates/images/2017/07/media-internals.png"
+       alt="Log panel in the chrome://media-internals page">
+  <figcaption>
+    <b>Figure 1.</b>
+    Log panel in the <i>chrome://media-internals</i> page
+  </figcaption>
+</figure>
+
+For those who want to understand what is happening, here's some JavaScript code
+snippet that shows you what Chrome is roughly doing behind the scenes.
+
+    var video = document.querySelector('video');
+    var selectedVideoTrackIndex;
+
+    document.addEventListener('visibilitychange', function() {
+      if (document.hidden) {
+        // Disable video track when page is hidden.
+        selectedVideoTrackIndex = video.videoTracks.selectedIndex;
+        video.videoTracks[selectedVideoTrackIndex].selected = false;
+      } else {
+        // Re-enable video track when page is not hidden anymore.
+        video.videoTracks[selectedVideoTrackIndex].selected = true;
+      }
+    });
+
+You may want to reduce the quality of the video stream when video track is
+disabled. It would be as simple as using the [Page Visibility API] as shown
+above to detect when page is hidden.
+
+And here are some restrictions:
+
+- This optimization only applies to videos with a [keyframe] distance < 5s.
+- If the video doesn't contain any audio tracks, the video will be
+  automatically paused when played in the background.
+
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=663999)
+
+## Automatic video fullscreen when device is rotated {: #auto-fullscreen-rotate }
+
+When a video is playing in the viewport, rotating the device to landscape
+orientation will now fullscreen the video. Rotating the device to portrait puts
+the video back to windowed mode as you would expect.
+
+Note that you can implement manually this behaviour yourself (see [Mobile Web Video
+Playback] article).
+
+<figure>
+  <img src="/web/updates/images/2017/07/auto-fullscreen-rotate.png"
+       alt="Automatic video fullscreen when device is rotated">
+</figure>
+
+This magic behaviour only happens when:
+
+- device is an Android phone (not a tablet)
+- user's screen orientation is set to "Auto-rotate"
+- video size is at least 200x200px
+- video uses native controls
+- video is currently playing
+- at least 75% of the video is visible (on-screen)
+- orientation rotates by 90 degrees (not 180 degrees)
+- there is no fullscreen element yet
+- screen is not locked using the Screen Orientation API
```

Comment:
Maybe consider linking to the spec. https://w3c.github.io/screen-orientation/

The API isn't documented on MDN. It looks like it shipped in the last version before we started contributing to MDN :(